latex/content/debug.tex

```diff
@@ -1,6 +1,6 @@
 \subsection{Debug}
 
-\begin{frame}[fragile]{\lstinline+\\overfullrule+}
+\begin{frame}[fragile]{\texttt{\textbackslash overfullrule}}
```

Comment:
Ah, ok. So it's the font color. Yes, I would prefer to change the appearance of lstinline in titles. Let me check how to do it.